src/main/scala/firrtl/transforms/CheckCombLoops.scala

```diff
@@ -207,8 +223,14 @@ class CheckCombLoops extends Transform {
         errors.append(new CombLoopException(m.info, m.name, expandedCycle))
       }
     }
+    val MN = ModuleName(c.main, CircuitName(c.main))
+    val annos = simplifiedModuleGraphs(c.main).getEdgeMap.toSeq.filter( _._2.nonEmpty ).map { edge =>
```

Comment:
I think this would be a little more clear (and remove a 2 unnecessary traversals) as:
```scala
val annos = simplifiedModuleGraphs(c.main).getEdgeMap.collect { case (from, tos) if tos.nonEmpty => ...
```